src/descriptor.ts

```diff
@@ -4,6 +4,7 @@ import * as type from "./type";
 import * as ts from "typescript";
 import * as comment from "./comment";
 import * as op from "./option.js";
+import * as pb from "google-protobuf";
```

Comment:
Why do we need this import here?

Reply:
The import is here so that the ugly `**pb.**Message.getField(fieldDescriptor, number)` calls can be made.  I think we have to keep using those ugly calls until we add _the has_field_name_ properties to the classes generated for protobuf messages (which would be done in a separate pull request).

Reply:
Ah I see.